npm/private/npm_import.bzl

```diff
@@ -615,6 +617,11 @@ def _impl_links(rctx):
             msg = "lifecycle_hooks_env contains invalid key value pair '%s', required '=' separator not found" % env
             fail(msg)
 
+    well_known_tags = propagate_well_known_tags(rctx.attr.lifecycle_hooks_execution_requirements)
```

Comment:
don't think we need to go through propagate_well_known_tags here since the property is execution_requirements and not tags the user is setting

Reply:
Thank you for the review, so I just read the doc here https://github.com/aspect-build/bazel-lib/blob/main/docs/run_binary.md, it seems the `execution_requirements` accept the `common args` defined here https://docs.bazel.build/versions/main/be/common-definitions.html#common.tags, so you mean we don't need to filter out the values are not belong to the `common args`?

Reply:
Correct. Since we're not pulling these from `tags` where there might be values that are not meant for execution requirements then we don't need to filter anything. The user knows that execution requirements values are expected.

Reply:
Got it!